com.unity.shadergraph/Documentation~/Upgrade-Guide-10-0-x.md

```diff
@@ -4,6 +4,10 @@
 
 Shader Graph has renamed the **Vector 1** property as **Float** in both the Vector 1 node and the exposed parameter list. The **Float** precision was also renamed as **Single**. Behavior is exactly the same, and only the names have changed.
 
+## Renamed Sample Cubemap
+
+Shader Graph has renamed the previous Sample Cubemap Node in Sample Reflected Cubemap and have added a new Sample Cubemap which used world space direction.
```

Comment:
little grammar fix here, 
"Shader Graph has renamed the previous Sample Cubemap Node **to** Sample Reflected Cubemap**,** and **has** added a new Sample Cubemap**,** which **uses** world space direction."

Reply:
thanks fixed